src/plugin/ipc/ssh/ssh.cpp

```diff
@@ -17,6 +17,8 @@
 using namespace dmtcp;
 
 #define SSHD_PIPE_FD -1
+#define ENV_ORIG_DPP "DMTCP_ORIGINAL_PATH_PREFIX"
```

Comment:
Could you add a comment above this:
```
// DPP, below, stands for "DMTCP Path Prefix"
```
@rohgarg seems to have invented this abbreviation.  But it was non-obvious to me.  This may also help others in reading the code.  Thanks.